frontends/infinite-corridor/src/test-utils/setupJest.ts

```diff
@@ -35,3 +35,8 @@ afterEach(() => {
   global.SETTINGS = _createSettings()
   resetApi()
 })
+
+const DEFAULT_DEVICE_WIDTH = "1200px"
+beforeAll(() => {
+  window.matchMedia = createMatchMediaForJsDom({ width: DEFAULT_DEVICE_WIDTH })
+})
```

Comment:
This lets us use MUI's `useMatchMedia` hook in JSDom tests. See docstring for `createMatchMediaForJsDom`